NEWS.md

```diff
@@ -104,6 +104,10 @@ This is a minor release and breaking changes have been kept to a minimum. End us
     `nlevel`, an alias for `scaled`, to better match the syntax of `stat_bin()`
     (@bjreisman, #2679).
 
+*   New theme elements allowing different ticks lengths for each
+    axis. For instance, this can be used to have inwards ticks on the
+    x-axis (`axis.ticks.length.x`) and outwards ticks on the y-axis
+    (`axis.ticks.length.y`). (@pank, #2935)
```

Comment:
Sorry to nitpick, one last request, then I'll merge: Please move the period to after `(@pank, #2935)`, just like in the other news bullets.